sds_data_manager/lambda_code/SDSCode/upload_api.py

```diff
@@ -12,13 +12,10 @@
 def _load_allowed_filenames():
     """
     Load the config.json file as a python dictionary.
-
```

Comment:
There should be a blank line here before you get to the "return" section I think, same for all of these below. They shouldn't be squashed together.

Reply:
Yeah sorry I need to re-format this section, I'm not sure why it got all weird